### PR TITLE
multiple_promises exercise solution fix

### PR DESCRIPTION
**Pull request by dammafra**
This is my implementation of multiple_promises solution, of which previous one was incorrect.
Only tested on OSX.

### CODE REVIEW

File: exercises/multiple_promises/solution/solution.js [hunk 1]
```diff
@@ -1,31 +1,33 @@
...
-  return new Promise(function (fulfill, reject) {
     var counter = 0;
     var out = [];
 
-    a.then(function (val) {
-      out[0] = val;
-      counter++;
+    promise1.then((value) => {
+        counter++;
+        arr.push(value);
```
Comment (AnneKlapwijk) [inline exercises/multiple_promises/solution/solution.js]: By using `array.push(value)`, `out` will contain values in order of when the promises are called. So lets say, `promise1` is called earlier than `promise2`, the result will be `[value1, value2]`, but if `promise2` is called earlier the result will be `[value2, value1]`. Wouldn't it be more convenient if the order of the values of `out` was corresponding to the arguments order of `all`?

File: exercises/multiple_promises/solution/solution.js [hunk 2]
```diff
@@ -1,31 +1,33 @@
-'use strict';
+function all(promise1, promise2) {
+    var fulfill, reject;
 
-/* global getPromise1, getPromise2 */
+    var p = new Promise((ff, rj) => {
```
Comment (AnneKlapwijk) [inline exercises/multiple_promises/solution/solution.js]: Why is this needed?

**Review by AnneKlapwijk — COMMENTED** [review]
I don't think the multiple_promises solution is incorrect as it is now. You were probably just returning the values in a different order.

**Comment by gplusdotgr** [comment]
@AnneKlapwijk is this repo still maintained? Would you like to pass it on?
